source/administration/command-line-tools.rst

```diff
@@ -680,6 +741,170 @@ mattermost export schedule
+------------------------
+
+  Description
+    Commands for managing Mattermost groups linked to a channel.
+
+  Child Commands
+    -  `mattermost group channel enable`_ - Enables group constraint on the specified channel
+    -  `mattermost group channel disable`_ - Disables group constraint on the specified channel
+    -  `mattermost group channel list`_ - Lists the groups associated with a channel
+    -  `mattermost group channel status`_ - Shows the group constraint status of the specified channel
+
+mattermost group channel enable 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Enables group constraint on the specified channel.  When a channel is group constrained, channel membership is managed by linked groups instead of managed by manually adding and removing users.
+
+  Format
+    .. code-block:: none
+
+      mattermost group channel enable {team}:{channel}
+
+  Examples
+    .. code-block:: none
+
+      ./mattermost group channel enable myteam:mychannel
+      
+mattermost group channel disable 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Disables group constraint on the specified channel.
+
+  Format
+    .. code-block:: none
+
+      mattermost group channel disable {team}:{channel}
+
+  Examples
+    .. code-block:: none
+
+      ./mattermost group channel disable myteam:mychannel
+      
+mattermost group channel list 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Lists the groups associated with a channel.
+
+  Format
+    .. code-block:: none
+
+      mattermost group channel list {team}:{channel}
+
+  Examples
+    .. code-block:: none
+
+      ./mattermost group channel list myteam:mychannel
+
+      
+mattermost group channel status 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Shows the group constraint status of the specified channel.  Returns "Enabled" when channel membership is managed by linked groups.  Returns "Disabled" when the channel membership is managed by manually adding and removing users. 
+
+  Format
+    .. code-block:: none
+
+      mattermost group channel status {team}:{channel}
+
+  Examples
+    .. code-block:: none
+
+      ./mattermost group channel status myteam:mychannel
+      
+mattermost group team 
+------------------------
+
+  Description
+    Commands for managing Mattermost groups linked to a team.
+
+  Child Commands
+    -  `mattermost group team enable`_ - Enables group constraint on the specified team
+    -  `mattermost group team disable`_ - Disables group constraint on the specified team
+    -  `mattermost group team list`_ - Lists the groups associated with a team
+    -  `mattermost group team status`_ - Shows the group constraint status of the specified team
+
+mattermost group team enable 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Enables group constraint on the specified team.  When a team is group constrained, team membership is managed by linked groups instead of managed by manually inviting and removing users.
```

Comment:
nit: double space

```suggestion
    Enables group constraint on the specified team. When a team is group constrained, team membership is managed by linked groups instead of managed by manually inviting and removing users.
```

Reply:
And same as in the `channel enable`, should we mention that an associated group is a requirement?

Reply:
Added a note to the config